src/main/java/abuhurairah/AbuHurairah.java

```diff
@@ -0,0 +1,47 @@
+package abuhurairah;
+
+import java.util.Scanner;
+
+/**
+ * The main class for the AbuHurairah task management application.
+ * This program stores and retrieves tasks, allowing users to interact
+ * via a command-line interface.
+ */
+public class AbuHurairah {
+    public static void main(String[] args) {
+        Ui ui = new Ui();
+        Storage storage;
+        Parser parser;
+        String name = "Abu Hurairah";
+
+        storage = new Storage("./AbuHurairahHistory.txt");
+        parser = new Parser();
```

Comment:
It would be more readable to initialize `storage` and `parser` in the same lines where you declared them.